gigascience/gigadb-website
# Improve accessibility of About Page

# Pull request for issue: #1368 

This is a pull request for the following functionalities:

* Heading levels not skipped
* Hero image has alt attribute
* Logo list uses a list instead of a table
* Logo images have alt attribute
* Logo links have accessible name
* Use page title to label section containing page content
* Group logo list inside main section since it is directly related to "indexing" text

## How to test?

* Visit site/about page, run WAVE audit ([chrome extension is the easiest way](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)) and confirm these specific issues are fixed (color contrast issues and common elements fixed in separate tickets) 

## How have functionalities been implemented?

* a new logolist less module defines styles for the list of logos using a flexbox and constaining size of images within
* Heading changed from h4 to h1. Decoupling heading levels with styles with class selector and variables which do not mention heading level

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
